pkg/globalconfig/global_config.go

```diff
@@ -229,7 +221,13 @@ func WriteGlobalConfig(config GlobalConfig) error {
 	if err != nil {
 		return err
 	}
-	cfgbytes, err := yaml.Marshal(config)
+
+	cfgCopy := config
+	// Remove some items that are defaults
+	if cfgCopy.RequiredDockerComposeVersion == RequiredDockerComposeVersionDefault {
+		cfgCopy.RequiredDockerComposeVersion = ""
+	}
```

Comment:
I'd avoid this completely. Instead use a getter function which returns a sane default if the config value is empty. Similar you did with the http ports above.

Reply:
Oh I see the function bellow which is used in the code. There you also could check for a value in the global config or use the default if empty.

Reply:
Instead of this code, better fix it on loading.

Reply:
Yes, that's clear, the default value should never be written to the file. But still I think validation and removal should better be done after reading and a getter function should take care to return the default value. Anyway, that's not a stopper, go ahead.